### DISCUSSION TITLE
support for [jasmine] 4.3.0

### DISCUSSION THREAD

**Discussion by koswedo** [DefinitelyTyped/DefinitelyTyped#61476] [Issues with a @types package] [answered]
**spyOnGlobalErrorsAsync** has been added in jasmine 4.3.0. Could it be added in here too?

Thank you

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
PR is ready https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61849
